app/src/main/java/com/gazlaws/codeboard/CodeBoardIME.java

```diff
@@ -67,6 +67,12 @@ public class CodeBoardIME extends InputMethodService
     private KeyboardUiFactory mKeyboardUiFactory = null;
     private KeyboardLayoutView mCurrentKeyboardLayoutView = null;
 
+    /* Added by Dimitris Stoukogiannis (aka ShadyGeek) on 14/2/2023.
+        This boolean variable is used to check whether the space button is long pressed or not.
+            If the value is false it means that the user hasn't long pressed on the space button, else it means he has.
```

Comment:
Nit: I also generally recommend using a gender neutral term ("they" instead of "he") for the user.